src/pilot/assign_role.py

```diff
@@ -781,7 +785,12 @@ def configure_raid(ironic_client, node_uuid, role, os_volume_size_gb,
         role, drac_client)
 
     if target_raid_config is None:
-        return False
+        return None
```

Comment:
While the above line would work, I think it would be better to stick with a boolean and return False.

Reply:
Done.